### PR TITLE
Add the new tab-less design

### PR DESCRIPTION
**Pull request by malcolmbaig**
screen shot 2016-06-01 at 13 44 51

### CODE REVIEW

**Comment by Davidslv** [comment]
is it possible to have a screenshot to see the look & feel ?

File: app/assets/sass/design-tabless.scss [hunk 1]
```diff
@@ -0,0 +1,1035 @@
...
+}
+
+.sibling-block-rhs ul a:after {
+  position: absolute;
+  top: 50%;
+  margin-top: -8px;
+  right: 10px;
+  float: right;
+  /*
+    content: "\203A";
```
Comment (Davidslv) [inline app/assets/sass/design-tabless.scss:332]: should this be removed?

Comment (malcolmbaig) [inline app/assets/sass/design-tabless.scss:332]: Probably, but i'd suggest largely ignoring the css for now. It's very much in flux and we've deliberately not worried too much about architecting it cleanly.

**Comment by malcolmbaig** [comment]
@Davidslv yep, done.

**Comment by Davidslv** [comment]
👍

**Pull request merged**